examples/full-example.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "My first Module",
+    "archiveName": "mycompany/my-first-module",
+    "sourceDir": "new_files",
```

Comment:
Maybe we should use "src" already, even if "new_files" is still often used. But I think the default should be "src".

Reply:
Great! I also prefer `src`.